datafusion/core/src/datasource/physical_plan/parquet/row_filter.rs

```diff
@@ -312,8 +369,10 @@ fn size_of_columns(
     Ok(total_size)
 }
 
-/// For a given set of `Column`s required for predicate `Expr` determine whether all
-/// columns are sorted. Sorted columns may be queried more efficiently in the presence of
+/// For a given set of `Column`s required for predicate `Expr` determine whether
+/// all columns are sorted.
+///
+/// Sorted columns may be queried more efficiently in the presence of
 /// a PageIndex.
 fn columns_sorted(
```

Comment:
This is interesting that we never connected up the "columns_sorted" information -- is this on your list @thinkharderdev ?

Should I file a ticket to do this?